pytest_tests/tests/payment/test_container_payments.py

```diff
@@ -119,7 +119,7 @@ def test_container_payments(
                     last_balance = current_balance
             assert len(deltas) == 1, "invalid number of withdrawals from the user wallet per epoch"
             single_node_gain_per_epoch = int((objects_count * MAX_OBJECT_SIZE) / GB)
-            assert abs(deltas[0] - (single_node_gain_per_epoch * replicas_number)) <= 0.5, "Invalid user wallet balance"
+            assert abs(deltas[0] - (single_node_gain_per_epoch * replicas_number)) <= 1, "Invalid user wallet balance"
```

Comment:
What's the expected value, what's happening in reality, why 1 is better than 0.5? All of this is expected in commit message.